site/app/templates/AddTerm.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}add-new-term{% endblock %}
+{% block title %}Add new Term{% endblock %}
+{% block form %}
+    <form method="post" action="{{ add_term_url }}">
+        {{ parent() }}
+    </form>
+{% endblock %}
+{% block body %}
+    <p class="std-margin">
+        A message is to be displayed here...
+    </p>
+    <p class="std-margin">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam tempor sed velit nec porttitor. Maecenas eu dolor est. Sed sit amet lacus sed elit accumsan molestie ullamcorper ut est. Curabitur molestie quam sit amet ex sodales venenatis. Proin porttitor erat vitae justo pharetra, id hendrerit felis sagittis.
```

Comment:
Before this pr is merged, this should be switched to real text or just removed

Reply:
it should be replaced with real text but i don't know what exactly should be written, any suggestions?